aries-site/src/pages/design-tokens/overview.mdx

```diff
@@ -0,0 +1,85 @@
+import { Box, Text } from 'grommet';
+import { Example } from '../../layouts';
+import { BetaNotification } from '../../examples';
+
```

Comment:
Consider the following as alternate approach from intro to "types":

```
#Overview

Design tokens allow HPE digital experiences to be presented as a unified, one-HPE. Tokens establish consistency by serving as a single source of truth encoding HPE's visual language. They are delivered in an easy-to-consume manner allowing the visual style to fluidly evolve along with HPE and customer needs.


##Visual language source of truth (or, Encoding HPE’s visual language)

Design tokens codify all decisions making up a design system’s visual style. Tokens represent colors, typography, spacing, motion, and more through clear names specifying how and where the token should be applied.  Consuming token names, as opposed to the values to which they resolve, builds inherit agility and flexibility into the system. Underlying values can be updated at the source of truth — and then seamlessly consumed by teams with little-to-no effort on their part. Simply point to the latest version and the updated styles are available in designs, sites, and applications. This means HPE digital experiences can be elevated more rapidly as a unified one-HPE. Change can be easy!


###What is a design token?

Tokens represent color, typography, spacing, motion, and other style values by a clear name conveying its intended usage.

An HPE design token consists of: | Property | Description | Example |
| ————- | —————— | ————- | | Name     | A self-documenting name representing an associated value. | `hpe.color.brand`, `example of a background token` | | Value     | The CSS(?) value to which is resolved in code. Alternatively, a value may reference another token. | `#01A982`, `example of a background token’s value` | | Type.     | A category for the token’s value. | Color, Dimension, Shadow | | Description | Concise text describing how the token is intended to be used. | |  Token names are purpose-driven rather than hard-coded values. This provides flexibility for HPE’s visual style to evolve over time, while minimizing impact on delivery teams. As example, perhaps the value for `example of background token` needs to be modified. The associated CSS value may be updated at the source of truth. All teams consuming `example of background token` automatically receive the updated value simply by referencing the latest version. No files touched, no find and replace — just an update pointing to a versioned release.


##Types of tokens (or, Token types, or Types of design tokens, …)

Possible minor tweaks to current content.
```

Reply:
Trying to hit the same points but adjust for order and hopefully more economical delivery.

Reply:
I like it, have incorporated and just replaced the "!" with a "." because it didn't feel like it fits with the broader style of the site.

Reply:
Agree. Fine with removing all together.

Reply:
Sounds good, I've removed the "Change" sentence.